fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -25,179 +40,112 @@ NONE_METHOD_OPT="NONE"
+opts_SetScriptDescription "Run fMRIVolume processing"
 
-script_name=$(basename "${0}")
-
-show_usage() {
-	cat <<EOF
+opts_AddMandatory '--studyfolder' 'Path' 'path' "folder containing all subject" "--path"
 
-${script_name}: Run fMRIVolume processing pipeline
+opts_AddMandatory '--subject' 'Subject' 'subject ID' ""
 
-Usage: ${script_name} [options]
+opts_AddMandatory '--fmritcs' 'fMRITimeSeries' 'file' 'input fMRI time series (NIFTI)'
 
-  [--help] : show usage information and exit
-  --path=<path to study folder>
-  --subject=<subject ID>
-  --fmritcs=<input fMRI time series (NIFTI)>
-  --fmriname=<name (prefix) to use for the output>
-  --fmrires=<final resolution (mm) of the output data>
+opts_AddMandatory '--fmriname' 'NameOffMRI' 'string' 'name (prefix) to use for the output'
 
-  --biascorrection=<method to use for receive coil bias field correction>
+opts_AddMandatory '--fmrires' 'FinalfMRIResolution' 'number' 'final resolution (mm) of the output data'
 
-        "SEBASED"
-             use bias field derived from spin echo images, must also use --dcmethod="${SPIN_ECHO_METHOD_OPT}"
+opts_AddMandatory '--biascorrection' 'BiasCorrection' 'SEBASED OR LEGACY OR NONE' "SEBASED: use bias field derived from spin echo images, must also use --dcmethod='${SPIN_ECHO_METHOD_OPT}'
 
-        "LEGACY"
-             use the bias field derived from T1w and T2w images, same as was used in 
-             pipeline version 3.14.1 or older. No longer recommended.
+LEGACY: use the bias field derived from T1w and T2w images, same as was used in pipeline version 3.14.1 or older (No longer recommended) 
 
-        "NONE"
-             don't do bias correction
+NONE: don't do bias correction"
 
-  [--fmriscout=<input "scout" volume>]
+opts_AddOptional '--fmriscout' 'fMRIScout' 'volume' "Used as the target for motion correction and for BBR registration to the structurals.  In HCP-Style acquisitions, the 'SBRef' (single-band reference) volume associated with a run is   typically used as the 'scout'. Default: 'NONE' (in which case the first volume of the time-series is extracted and used as the 'scout')  It must have identical dimensions, voxel resolution, and distortions (i.e., phase-encoding   polarity and echo-spacing) as the input fMRI time series" "NONE"
 
-      Used as the target for motion correction and for BBR registration to the structurals.
-      In HCP-Style acquisitions, the "SBRef" (single-band reference) volume associated with a run is 
-      typically used as the "scout".
-      Default: "NONE" (in which case the first volume of the time-series is extracted and used as the "scout")
-      It must have identical dimensions, voxel resolution, and distortions (i.e., phase-encoding 
-      polarity and echo-spacing) as the input fMRI time series
+opts_AddOptional '--mctype' 'MotionCorrectionType' 'MCFLIRT OR FLIRT' "What time of motion correction to use MCFLIRT or FLIRT" "MCFLIRT"
```

Comment:
Typo, should be "What type".  Also, please indicate what the default is.

Reply:
Done